docs/deploy-production.md

```diff
@@ -60,21 +60,21 @@ The same dockerfile describe above is used to host a ipfs node. Side note, if yo
 
     git clone https://github.com/AugurProject/augur.git
     cd <augur repository>
-    docker build .
+    docker build . -t augurclient
 
 After docker image has been built successfully, check for message `Successfully built <image hash>`, run it with -p to map port to the internal docker container port, ipfs swarm will serve up files via port 4001. Optionally port 8001 uses nginx to pass requests to ipfs gateway hosted in the docker container. 
 
-    docker run -p 4001:4001 -p 8001:8001 <image hash>
```

Comment:
Was the `-t` necessary here or was it a copypasta from `docker build`

Reply:
Also curious about this, do we need to include -t augurclient any time this image is used?

Reply:
true '-t' isn't needed (copy paste issue), it's a valid flag but for `Allocate a pseudo-TTY` from the documentation. confusing I'll remove it.